lib/conjur/version.rb

```diff
@@ -19,6 +19,6 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 module Conjur
-  VERSION = "4.29.0"
+  VERSION = "4.29.1"
```

Comment:
When adding new features, the second number needs to be incremented. You can check https://rubygems.org/gems/conjur-cli to see what version is currently released. (It's 4.28.2). So we can keep adding features to 4.29.0. Incrementing the patch version would only be done if the gem was already released and we had to fix a bug.